youtube_dl/extractor/brightcove.py

```diff
@@ -508,6 +510,20 @@ def _extract_urls(webpage):
                 'http://players.brightcove.net/%s/%s_%s/index.html?videoId=%s'
                 % (account_id, player_id, embed, video_id))
 
+        # 	<video data-video-id="5320421710001" data-account="245991542" data-player="SJWAiyYWg" data-embed="default" class="video-js" controls itemscope itemtype="http://schema.org/VideoObject">
+        for video in re.findall(r'(?i)(<video[^>]+>)', webpage):
+            attrs = extract_attributes(video)
+                
+            video_id    = attrs.get('data-video-id')
+            account_id  = attrs.get('data-account')
+            player_id   = attrs.get('data-player')
+            embed       = attrs.get('data-embed')
+
+            if video_id and account_id and player_id and embed:
+                entries.append(
+                    'http://players.brightcove.net/%s/%s_%s/index.html?videoId=%s'
+                    % (account_id, player_id, embed, video_id))
+
```

Comment:
I was talking about the video on programme-tv.net. That's unrelated to bostonglobe.com.

> That doesn't work for the bostonglobe.com case

Of course it won't as these lines are for programme-tv.net

> and then bostonglobe.py should import that

Nope. bostonglobe.py requires special handling and is unrelated to _extract_urls()

> As a side note, this pattern is quite similar to #12005

That's just a reminder for myself but not requiring you to implement both patterns in a single pull request. I'm sorry if that's misleading. And apparently it's time to seperate codes for bostonglobe.com and programme-tv.net into two pull requests. Otherwise there might be more confusions.